js/examples.js

```diff
@@ -0,0 +1,4 @@
+const EXAMPLES = {
+  "hello":"Tools.renderTweets(['Hello, world!']);",
+  "markov":'var corpus = $("#markov textarea").val();\noutput = Tools.markovTrainAndGenerate(corpus);\nTools.renderTweets(output);',
```

Comment:
This should probably do URL loading instead of just putting code literals in here so we don't have to type non-trivial examples this way. I'll fix it.